### PR TITLE
ADR: Deploy Strategy Change

### PR DESCRIPTION
**Pull request by carltonsmith**
### Summary of Changes

Documentation of deploy strategy change proposal

### CODE REVIEW

**Review by kniz-raft — APPROVED** [review]
Looks good to me

**Review by jtwillis92 — APPROVED** [review]

File: docs/Architecture Decision Record/014-deploy-strategy.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Proposed Decision
+
+We would like to replace the Government and Vendor staging sites with one staging site that will live in the `tanf-staging` Cloud.gov space.
+
+_Note that this represents a planned future state, not the current state._
+
+Deploy Environment | Cloud.gov Space | Cloud.gov Dev Access | Role                                              | Deploys when ...  |
+-------------------|-----------------|----------------------|---------------------------------------------------|-------------------|
+Development        | Dev             | Vendor & Gov         | Deploy code for testing & review  | Anytime           |
+Staging        | Staging             | Vendor & Gov         | Deploy code after gov review            | Code merged to `raft-tech/TANF-app:raft-tdp-main` | `HHS/TANF-app:staging`
```
Comment (ADPennington) [inline docs/Architecture Decision Record/014-deploy-strategy.md:22]: can we confirm that no PII will be in staging?

Comment (carltonsmith) [reply docs/Architecture Decision Record/014-deploy-strategy.md:22]: Yes, there are no plans to use PII in the staging environment. It's just the final place to test approved code.

**Comment by alexsoble** [comment]
@carltonsmith @ADPennington Notes from Tech Sync: 

* We'll want to update this line in our [boundary diagram documentation](https://github.com/HHS/TANF-app/blob/bd4127dd1d69a649409675ae9410238056c5f8fa/docs/Security-Compliance/boundary-diagram.md):
> Developer access to the Dev space does not permit access to Staging and Production. 
* We can explicitly mention that there will be no PII in the Staging space as well.   
* We'll also want to make sure that any references like this in the SSP are updated.
* Finally, we could close out #723 once this is approved and merged ... and maybe make a stub ticket for setting up a prod app! 😄

**Comment by carltonsmith** [comment]
> @carltonsmith @ADPennington Notes from Tech Sync:
> 
> * We'll want to update this line in our [boundary diagram documentation](https://github.com/HHS/TANF-app/blob/bd4127dd1d69a649409675ae9410238056c5f8fa/docs/Security-Compliance/boundary-diagram.md):
> 
> > Developer access to the Dev space does not permit access to Staging and Production.
> 
> * We can explicitly mention that there will be no PII in the Staging space as well.

@alexsoble I have updated the boundary diagram documentation accordingly

**Review by ADPennington — APPROVED** [review]
👍🏾 appreciate these updates @carltonsmith. 

cc: @lfrohlich @alexsoble

File: docs/Architecture Decision Record/014-deploy-strategy.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# 14. Deployment Strategy Change
+
+Date: 2021-04-06 (yyyy-mm-dd)
+
+## Status
+
+Pending
```
Comment (ADPennington) [inline docs/Architecture Decision Record/014-deploy-strategy.md:7]:
```suggestion
Approved
```

**Pull request merged**